core/src/main/java/com/yahoo/oak/NativeMemoryAllocator.java

```diff
@@ -97,7 +97,7 @@ public boolean allocate(Slice s, int size, MemoryManager.Allocate allocate) {
 
                 // We read again the buffer so to get the per-thread buffer.
                 // TODO: This will be redundant once we eliminate the per-thread buffers.
-                readByteBuffer(s);
+                //setSliceBuffer(s);
```

Comment:
Yes! This should have been removed in earlier PRs. The comment can be removed as well.

Reply:
done